tasks.py

```diff
@@ -26,13 +17,23 @@ def deploy_nixos(hosts: List[DeployHost]) -> None:
     g = DeployGroup(hosts)
 
     def deploy(h: DeployHost) -> None:
-        target = f"{h.user or 'root'}@{h.host}"
         h.run_local(
-            f"rsync {' --exclude '.join([''] + RSYNC_EXCLUDES)} -vaF --delete -e ssh . {target}:/etc/nixos"
```

Comment:
I did this mainly for speed because the remote machines is at least as powerful as my local machine and this way I don't have to upload packages to the remote builder.
Can you elaborate on why you want to switch to `nixos-rebuild`?

Reply:
Want to switch away from rsync and needing to keep an exclude list for it, could switch to something else beside nixos-rebuild?

Reply:
How about this alternative? That wouldn't need maintaining a filter.

```
rsync "$(nix flake info --json | jq -r .path)" ...
```

Reply:
How do you like this? https://github.com/nix-community/infra/pull/460

Reply:
That's good, thank you.